docs/application/vscode-ext/web.md

```diff
@@ -79,142 +51,158 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
   >
   > Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
 
-  1. Open **Command Palette** and enter **Run Emulator-Manager**.
-  2. Select the profiles and versions that you want to install and click **Ok**.
+  1. In VS Code, click **View** > **Command Palette**.
+  2. In the input field of the **Command Palette** that appears, type **Run Emulator-Manager**, and press **Enter**.
+  3. Select the profiles and versions for your project and click **install**.
 
      ![Emulator Manager](media/howtoinstall-emulatormanager.png)
 
 ## Develop Applications
 
-The following section explains how to use VS Code Extension for Tizen Web to develop the applications:
+This section explains how to use VS Code Extension for Tizen Web to develop applications.
 
 ### Create Tizen Web Project
 
-To create a Tizen Web project, follow these steps:
+- Open **Command Palette** and enter **Create Wizard Project**. Choose, whether you want to use an existing Tizen Baseline SDK installation or install a new one
+
+  ![Baseline SDK installation query](media/tizen_web5.png)
+
+  - To perform a fresh installation, click **YES**.
+
+    To perform a fresh installation and to begin the Tizen Baseline SDK installation, click **YES**. Else, click **NO**. If **NO** is clicked, it is assumed that you have already done the installation and the project is created.
+
+- To create a Tizen Web project, follow these steps:
 
-1. Open **Command Palette** and enter **Create Wizard Project**.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Create Wizard Project**, and press **Enter**.
+3. In the notification that appears, to perform a fresh installation and to begin the Tizen Baseline SDK installation, click **YES**. otherwise, click **NO**. If **NO** is clicked, it is assumed that you have already done the installation and the project is created.
```

Comment:
hm.. I don't understand why add semi colon.
but, I will add your comment :(

done.

Reply:
@DohyungLim,

The reason to add a semicolon is that it's often a good idea to set off certain kinds of clauses with a comma or other punctuation, for clarity.
As per Google Style guide: In general, put a semicolon or a period or a dash before a conjunctive adverb, such as "otherwise" or "however" or "therefore," and put a comma after the conjunctive adverb.

For more information, see https://developers.google.com/style/commas